### PR TITLE
fix: update diff link to include dash between versions and features

### PR DESCRIPTION
**Pull request by andreifilip123**
Closes #<issue>

## ✅ Checklist

- [ ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [ ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

**Comment by juliusmarminge** [comment]
This seems to 404 for every single one

**Comment by andreifilip123** [comment]
Looking some more into this but I noticed that some diffs are actually missing. That is because of the bug with 0's in version numbers. Fixed that in create-t3-app-diffs and now generating the diffs (22k)

File: upgrade/src/app/not-found.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            404 - Page Not Found
+          </p>
+          <p>
+            Sorry about that. This diff seems to be missing. Come back later or
+            try a different version.
+          </p>
+        </div>
+
+        <p className="text-center text-2xl font-extrabold tracking-tight hover:underline sm:text-[2rem]">
+          <Link href="/">Go Home</Link>
```
Comment (juliusmarminge) [inline upgrade/src/app/not-found.tsx:21]:
```suggestion
<a href="/">Go Home</a>
```

File: upgrade/src/app/not-found.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+
```
Comment (juliusmarminge) [inline upgrade/src/app/not-found.tsx:2]:
```suggestion

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**